uisdk/src/androidTest/java/com/karhoo/uisdk/common/BaseRobot.kt

```diff
@@ -349,25 +344,6 @@ open abstract class BaseTestRobot {
         }
     }
 
-    /**
```

Comment:
I think we should probably leave this here

Reply:
It creates an error. I'll try again just in case